### ISSUE TITLE
Don't use anilist

### ISSUE THREAD

**Issue by Cyliann** [justchokingaround/jerry#11]
Is it possible to skip anilist authorization and use local history only? I don't use anilist, but the script keeps asking me to enter anilist token. Can I somehow disable it?

**Comment by justchokingaround** [comment]
yes, you can alias jerry to `jerry -i` in your shell .rc file. i can add an option in the config file if you want? and i plan to add an ani-cli-like mode in the future anyway

**Comment by Cyliann** [comment]
But jerry -i won't save to my local history either, but I want to have local history

**Comment by justchokingaround** [comment]
ah, i see what you mean. this isn't my priority right now, but i will eventually implement it, a few commits after a i merge `feat-manga`

**Comment by ttx99** [comment]
im waiting on this also no anilist required.

**Comment by justchokingaround** [comment]
@Cyliann @ttx99 
this has now been implemented on the `dev` branch, from my testing so far everything seems to work as expected. you can either use the `--no-anilist` flag, or specify `no_anilist=1` in your `~/.config/jerry/jerry.conf`

if you need help with testing the version of the script that's on the `dev` branch etc. ask me and i'll help.

sorry for the delay on this :pensive: 

https://github.com/justchokingaround/jerry/commit/e3ac8da8cafd47b38a893375950a85cf1b511b2b

**Comment by Cyliann** [comment]
Thanks a lot. Since this then I starred using anilist, so I don't need this feature now, but I'm sure someone will need it.
I'm greatful for your work

**Comment by justchokingaround** [comment]
no problem, this was a planned feature anyways. i haven't really made a public announcement yet, nor documented most of the stuff, but i've recently fixed jerry on the main branch, it now uses 9anime as the default provider (also supports zoro and yugen), in case ure interested in still using it. again, sorry for the delay, was going through some rough times, i will now be actively maintaining the project

**Comment by justchokingaround** [comment]
merged into main, closing this now
